File: Python/tests/test_wcs_sql.py
import pytest

from hbayesdm.models import wcs_sql


def test_wcs_sql():
    _ = wcs_sql(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_wcs_sql.R
context("Test wcs_sql")
library(hBayesDM)

test_that("Test wcs_sql", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(wcs_sql(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
